downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md

```diff
@@ -43,6 +43,18 @@ A few backward-incompatible changes have been introduced during the Swan Lake Be
 
 ### New features
 
+#### Tuple member annotations 
+
+- Added support for annotating tuple members
+- Tuple members can be annotated using annotations with `field` attachment points
+- Similar to records, annotations are not allowed on the `rest` type of a tuple
```

Comment:
@ushirask, can you please update this to be in like a paragraph form. We don't usually use point form when describing a feature.

Reply:
Updated in #6320